File: pinyougou-parent/pinyougou-manager-web/src/main/webapp/js/service/brandService.js
app.service('brandService',function($http){
    //读取列表数据绑定到表单中
    this.findAll=function(){
        return $http.get('../brand/findAll.do');
    }
    this.findOne=function (id) {
        return $http.get('../brand/findOne.do?id='+id);
    }
    this.findPage=function (page,rows) {
        return $http.get('../brand/findPage.do?page=' + page + '&rows=' + rows);
    }
    this.dele=function (selectIds) {
        return $http.get('../brand/delete.do?ids='+selectIds);
    }
    this.search=function (page,rows,searchEntity) {
        return $http.post('../brand/search.do?page='+page+"&rows="+rows, searchEntity);
    }
    this.save=function (methodName,entity) {
        return $http.post('../brand/'+ methodName +'.do',entity );
    }
    this.selectOptionList=function () {
        return $http.get('../brand/selectOptionList.do');
    }
});

File: pinyougou-parent/pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import entity.PageResult;
import com.pinyougou.pojo.TbBrand;

import java.util.List;
import java.util.Map;

public interface BrandService {
    public List<TbBrand> findAll();
    public PageResult findPage(int pageNum, int pageSize);
    public void add(TbBrand brand);
    public void update(TbBrand brand);
    public TbBrand findOne(Long id);
    public void delete(Long [] ids);
    public PageResult findPage(TbBrand brand, int pageNum,int pageSize);
    List<Map> selectOptionList();
}
